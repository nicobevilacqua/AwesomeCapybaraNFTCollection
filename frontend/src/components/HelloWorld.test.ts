import { mount } from '@vue/test-utils';
import HelloWorld from './HelloWorld.vue';

describe('asd', () => {
  it('test', () => {
    const wrapper = mount(HelloWorld, {
      props: {
        msg: 'test',
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
