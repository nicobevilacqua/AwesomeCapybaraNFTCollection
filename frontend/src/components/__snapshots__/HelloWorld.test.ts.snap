// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asd test 1`] = `
"<h1>test</h1>
<p> Recommended IDE setup: <a href=\\"https://code.visualstudio.com/\\" target=\\"_blank\\">VSCode</a> + <a href=\\"https://github.com/johnsoncodehk/volar\\" target=\\"_blank\\">Volar</a></p>
<p>See <code>README.md</code> for more information.</p>
<p><a href=\\"https://vitejs.dev/guide/features.html\\" target=\\"_blank\\"> Vite Docs </a> | <a href=\\"https://v3.vuejs.org/\\" target=\\"_blank\\">Vue 3 Docs</a></p><button type=\\"button\\">count is: 0</button>
<p> Edit <code>components/HelloWorld.vue</code> to test hot module replacement. </p>"
`;
